widget/tabbar/xresources.lua

```diff
@@ -0,0 +1,58 @@
+local awful = require("awful")
```

Comment:
Looking really good. I think the name 'xresources' doesn't really fit though since it doesn't have anything more to do with xresource than the other styles. That might mislead users. Unfortunately, the name `default` is already taken which would have been perfect as it really blends in with the default awesomewm config. Maybe just take your username here or something else that comes to your mind.

Reply:
Maybe, vanilla or something but yes default would be perfect

Reply:
`pure` maybe?
I will rename, thank you guys